geode-core/src/main/java/org/apache/geode/internal/statistics/ValueMonitor.java

```diff
@@ -98,6 +98,9 @@ protected void monitorStatistics(final long millisTimeStamp,
         if (this.statistics.contains(resource.getStatistics())) {
           ResourceType resourceType = resource.getResourceType();
           StatisticDescriptor[] sds = resourceType.getStatisticDescriptors();
+          if (!resource.getStatValuesNotified()) {
+            resource.setStatValuesNotified(true);
+          }
```

Comment:
I think that the `if` is unnecessary here, since it's fine to call `setStatValuesNotified(true)` if the value is already true.